src/recipes/tooling-editor-plugin.md

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Editor Plugin
+
+Using `EditorPlugin` types is very similar to the process used when
+[writing plugins in GDScript](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/making_plugins.html). However,
+like Custom Classes, they
+are registered automatically by godot-rust and cannot be enabled/disabled in the Project Settings plugins pane.
```

Comment:
What do you mean with "Custom Classes" here?

Editor plugins _are_ custom classes. Maybe you're looking for a different term?